docs/get-started/quickstarts/collect-data.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Collect images or sensor data in 2 minutes"
+linkTitle: "Collect data in 2 minutes"
+type: "docs"
+weight: 20
+cost: 75
+images: ["/platform/data.svg"]
+description: "Use Viam to gather images or sensor data from your machine."
+---
+
+Follow this guide to start collecting images or sensor data from your machine using the [data management service](/services/data/):
```

Comment:
My thought was that people might read "this guide" and think we mean the linked data page instead of the page they're already on, but that's probably not evident anyway so I'll apply your suggestion!